vscode/src/completions/logger.ts

```
@@ -290,7 +295,7 @@ function lineAndCharCount({ insertText }: InlineCompletionItem): { lineCount: nu
 const TEN_MINUTES = 1000 * 60 * 10
 const errorCounts: Map<string, number> = new Map()
 export function logError(error: Error): void {
-    if (isAbortError(error) || isRateLimitError(error)) {
+    if (isAbortError(error) || isRateLimitError(error) || isAuthError(error)) {
```

Ignoring this error here means it also won't count for the sad user metric anymore (since that is derived from the analytics logging of errors) which mean we consider this to be a "good UX". I don't think it's true. If something is causing excessive auth errors for users to pop up, it's something we should fix, just like #[970](https://github.com/sourcegraph/cody/pull/970). I am not sure I agree that simply ignoring this is a good fix?

---

Mhh maybe until we have fixed the access token becoming invalid, this is a reasonable change. I just wonder if we should add some comment or other reminder that we clean this up again once we fix it so that an invalid auth token change automatically logs a user out of the client

---

@philipp-spiess, I created a follow-up issue that should fix the underlying problem. WDYT? We shouldn't have an excessive number of `not authenticated` requests if we log out users after the first one. 

- https://github.com/sourcegraph/cody/issues/1027

---

Sounds great!